src/Panel.php

```diff
@@ -60,7 +60,10 @@ public function getPanel()
 		$installed = $this->decode($lockFile);
 
 		if ($this->error === NULL) {
+			$required = array_filter($required);
+			$installed = array_filter($installed);
```

Comment:
Yes, a have in composer.lock "packages-dev": null
I have packages in require-dev section and call "composer update --no-dev". I try install newest version of composer (composer self-update)